packages/callout-block/src/CalloutBlock.tsx

```diff
@@ -33,19 +48,24 @@ export const CalloutBlock: FC<CalloutBlockProps> = ({ appBridge }) => {
         }
     };
 
-    const textDivClassNames = joinClassNames([
-        'tw-flex tw-items-center',
-        isDark(getBackgroundColor(blockSettings.type)) ? 'tw-text-white' : 'tw-text-black',
-        blockSettings.width === Width.FullWidth && alignmentMap[blockSettings.alignment],
-        !blockSettings.hasCustomPadding && paddingMap[blockSettings.paddingChoice],
-    ]);
-
     const customPaddingStyle = {
         padding: blockSettings.hasCustomPadding
             ? `${blockSettings.paddingTop} ${blockSettings.paddingRight} ${blockSettings.paddingBottom} ${blockSettings.paddingLeft}`
             : '',
     };
 
+    const color = getBackgroundColor(blockSettings.type);
+    const backgroundColor = blockSettings.appearance === Appearance.Strong ? setAlpha(color, 1) : setAlpha(color, 0.1);
```

Comment:
This way the color of the user, if has selected a alpha value, is overwritten. Is this correct? If not, you could do it this way and it would keep the users color:

```suggestion
    const backgroundColor = blockSettings.appearance === Appearance.Strong ? color : setAlpha(color, 0.1);
```

Reply:
Another thing. I think currently it's a little confusing that `getBackgroundColor(...)` returns something of type `string | undefined`, the parameter of setAlpha is typed as `color: unknown` and in its implementation we just assume its of type Color or ColorInput `isRgbaLongFormat(color as Color) ? toShortRgba(color) : (color as ColorInput)`

Reply:
you're right. I changed it to your suggestion - thanks 👍